src/index.html

```diff
@@ -11,12 +11,34 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img src="./images/logo.png"
+          alt="logo"
+          class="logo">
```

Comment:
```suggestion
        <img
          src="./images/logo.png"
          alt="logo"
          class="logo"
        >
```

Reply:
all done, could you check again, please. It still doesn't go through the test.